helix-core/src/main/java/org/apache/helix/manager/zk/client/HelixZkClient.java

```diff
@@ -35,8 +36,48 @@
 
   void unsubscribeDataChanges(String path, IZkDataListener listener);
 
+  /*
+   * This is for backwards compatibility.
```

Comment:
Nit: backward-compatibility

Reply:
Is `backwards compatibility` an incorrect usage? My understanding is `backwards` may look more like British style, and `backward` is USA style. Correct me if I am wrong.